src/Messages.jl

```diff
@@ -609,7 +620,7 @@ function Base.show(io::IO, m::Message)
     end
     println(io, typeof(m), ":")
     println(io, "\"\"\"")
-    writeheaders(io, m)
+    writeheaders(io, m; mask_authorization_header=true)
```

Comment:
Perhaps it is better to not modify the code of `writeheaders` and instead handle everything locally in this show method? Something like
```julia
iob = IOBuffer()
writeheaders(iob, m)
header_str = String(take!(iob))
header_str = replace(header_str, r"(*CRLF)^(Authorization: ).*$"m => s"\1******")
write(io, header_str)
```

Are there any other headers that may contain sensitive information that we should consider scrubbing as well?

Reply:
Agreed. I don't think we should touch `writeheaders` for a change to the `show` method.

> Are there any other headers that may contain sensitive information that we should consider scrubbing as well?

`Cookie`/`Set-Cookie`.

Reply:
Fixed in https://github.com/JuliaWeb/HTTP.jl/pull/1127/commits/1dd214ba07c93095a8d147c87a12680974c36e34